src/spectaql/augmenters.js

```diff
@@ -193,7 +194,10 @@ function hideFields(options = {}) {
     }
 
     // Handle OBJECT.fields AND INPUT_OBJECT.inputFields
-    for (const field of type.fields || type.inputFields || []) {
+    for (const field of type.fields ||
+      type.inputFields ||
+      type.enumValues ||
```

Comment:
This was part of the problem, but so was lack of solid support from `microfiber`...so I fixed that, too